op-node/node/config.go

```diff
@@ -32,7 +32,7 @@ type Config struct {
 
 	Rollup rollup.Config
 
-	DependencySet derive.DependencySet
+	DependencySet depset.DependencySet
```

Comment:
Wondering why this was switched to the supervisor `DependencySet` from the [simpler node `DependencySet` interface](https://github.com/ethereum-optimism/optimism/blob/ad8f15377ad6b65ac0b40d232cb18852c2b43177/op-node/rollup/derive/attributes.go#L16-L19)?  Seems like the rpc method would just need to expose the chain ids.

Reply:
The RPC really should expose the same depset.json as was originally fed to op-node. Technically it could still use the simpler derive.DependencySet and the actual implementation would still be the full thing but it seemed better to set the expectation that this was a full DependencySet.